javascripts/eventhandlers.js

```diff
@@ -0,0 +1,31 @@
+"use strict";
+var EventHandlers = (function(chip) {
+
+	chip.activateEvents = () => {
+		var master = $("#master-wrapper");
+		var input = $("input");
+
+		function keypress(keyin) {
```

Comment:
There is no recursion.  I just added an event listener to the whole page.  If I click on an item with class "col-sm-4 car", then I start the keyin listener that replaces text until I press enter